src/home/swap-container/components/swap/index.tsx

```diff
@@ -253,7 +254,12 @@ const Swap = ({
           </Typography>
         ),
       });
-      const result = await walletService.approveToken(from, !!(shouldEnableYield && fromYield?.tokenAddress));
+      const result = await walletService.approveToken(
```

Comment:
Could we also have a new TRANSACTION_TYPES that is 'APPROVE_TOKEN_EXACT' and we send both the token and the amount approved, so then we can later show both on the snackbar 
"Approve X <TOKEN> to be used" and on the wallet transaction timeline the same text